core/graphics.rb

```diff
@@ -111,4 +187,13 @@ def configure_gl_version version='3.2'
       Glfw::Window.window_hint(Glfw::OPENGL_PROFILE, Glfw::OPENGL_CORE_PROFILE)
     end
   end
+
+  def self.error_check
+    error = GL::glGetError()
+    if error != GL::GL_NO_ERROR
+      puts "GLError: #{error.to_s(16)}" 
+      puts caller
+    end
+#     raise "GLError: #{error.to_s(16)}" unless error == GL::GL_NO_ERROR
```

Comment:
I fixed the GLError problems I was having by using a different opengl32.dll, but while I was trying to figure out what was wrong, I told it to print instead.
I still don't know what was wrong other than that haha